src/components/Icon/Icon.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import { afterEach } from "vitest";
+
+import { Icon } from "./Icon";
+
+afterEach(() => {
+  cleanup();
+});
+
+describe("Icon component for CDC Component Library", () => {
+  it("should render an icon", () => {
+    const { container } = render(<Icon name="settings" />);
```

Comment:
Let's have a simple assertion here that the container has some expected value.

Reply:
I'm not certain what this means. May you elaborate?

Reply:
Use an `expect` function like you did in the other tests.  Can maybe expect that an `svg` tag exists.  Can be simple.